src/scripts/trust-dev-certs.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+#
+# Updates cacerts trust store in Java home to trust:
+#   The Lighthouse development certs, internal-sys-dev.
+#   The nexus.freedomstream.io certificate.
+#   The tools.health.dev-developer.va.gov certificate.
+#
+# You will probably need root access to modify the trust store.
+#
+
+remove_old_certificate () {
```

Comment:
maybe I just dont understand what this is doing.  But why do we need this script here?

Reply:
This question is about trust-dev-cert.sh in general

Reply:
This script is never used, as far as I've seen. I'm gonna delete it until someone complains that they actually really needed it for local dev (which is I think what it was originally here for)

Reply:
Turns out test-starter uses it for... something so we should look into why it's still considered important.